src/main/java/me/moodcat/api/models/RoomModel.java

```diff
@@ -0,0 +1,36 @@
+package me.moodcat.api.models;
+
+import java.util.List;
+
+import lombok.Data;
+import me.moodcat.database.entities.Room;
+import me.moodcat.database.entities.Song;
+
+/**
+ * The model of the room.
+ */
+@Data
+public class RoomModel {
+
+    /**
+     * The current song of the room model.
+     */
+    private Song currentSong;
+
+    /**
+     * The name of the room model.
+     */
+    private String name;
+
+    /**
+     * The constructor is used to create a room model given a {@link Room}.
+     *
+     * @param room
+     *            the room to create the room model of.
+     */
+    public RoomModel(Room room) {
+        this.currentSong = room.getCurrentSong();
```

Comment:
But my point is that if I want to run this branch with the frontend. I get errors that room.song isn't found due to the change from room.song to room.currentSong.